features/enlinkd/persistence/api/src/main/java/org/opennms/netmgt/enlinkd/persistence/api/IpNetToMediaDao.java

```diff
@@ -44,23 +44,23 @@
  */
 public interface IpNetToMediaDao extends OnmsDao<IpNetToMedia, Integer> {
     
-    public List<IpNetToMedia> findBySourceNodeId(Integer id);
+    List<IpNetToMedia> findBySourceNodeId(Integer id);
 
-    public List<IpNetToMedia> findByPhysAddress(String physAddress);
+    List<IpNetToMedia> findByPhysAddress(String physAddress);
 
-    public List<IpNetToMedia> findByNetAddress(InetAddress netAddress);
+    List<IpNetToMedia> findByNetAddress(InetAddress netAddress);
 
-    public IpNetToMedia getByNetAndPhysAddress(InetAddress netAddress, String physAddress);
+    IpNetToMedia getByNetAndPhysAddress(InetAddress netAddress, String physAddress);
 
     void deleteBySourceNodeIdOlderThen(Integer nodeiId, Date now);
     
-    public void deleteBySourceNodeId(Integer nodeId);
+    void deleteBySourceNodeId(Integer nodeId);
 
     /**
      * Finds all {@link IpNetToMedia} instances that have a physical address that is equal to the mac address
      * of a {@code BridgeMacLink} of the addressed {@code node}.
-     * @param nodeId
-     * @return
-     */
+     * @param nodeId "The nodeId"
```

Comment:
I woluld remove this line completely. It says exactly the same as in the code itself. I would only add a comment if it adds additional information.

Reply:
commit 309c8f3599c552bb4f571dd78dfb6e9e4922c5c8 (HEAD -> jira/ar/NMS-14434)
Author: Antonio <rssntn67@yahoo.it>
Date:   Mon Jun 27 10:16:16 2022 +0200

    NMS-14434 Enlinkd Clean Persistence Classes

    Resolve conversation patrick-schweizer
    Removed comment